docs/getting-started.rst

```diff
@@ -1,59 +1,112 @@
+---------------
 Getting started
 ---------------
 
-Requirements:
 
-- ``docker`` installed and running
-- ``Node.js`` >= 7.6 - ``async/await`` support
 
-1. Download repositories: gardener-server, gardener-smart-contracts
+
+Repositories
+------------
+
+Gardener contains three repositories, two main ones: gardener-smart-contracts which holds smart contracts, and gardener-server, which is responsible for fetching data from third party data sources.
+The third one, gardener-monitor is optional and it helps visualizing requests. You can easily clone them.
+
+1. Gardener server
+::
+
+  git clone https://github.com/EspeoBlockchain/gardener-server.git
+
+2. Gardener smart contract
+::
+
+  git clone https://github.com/EspeoBlockchain/gardener-smart-contracts.git
+
+3. Gardener monitor (optional)
 ::
 
-   git clone https://github.com/EspeoBlockchain/gardener-server.git
-   git clone https://github.com/EspeoBlockchain/gardener-smart-contracts.git
+  git clone https://github.com/EspeoBlockchain/gardener-monitor.git
 
 
-2. Copy smart contracts variables from template
+Running blockchain
+------------------
+
+Before we will get information from external sources to our blockchain, we have to run it first. Let's use ganache for it.
+Make sure that you are in gardener-server catalog, then:
 ::
 
-   cd gardener-smart-contracts
-   make copy-env
+  make ganache
 
-3. Copy server variables from template
+If you see this information:
 ::
 
-   cd ../gardener-server
-   make copy-env
+  Creating test-blockchain ... done
 
-4. Run test blockchain
+That means you have created test blockchain successfully. You can verify its status using:
 ::
 
-   make ganache
-   docker ps
+  docker ps
 
-5. Install smart contracts dependencies
+After that you need to copy our smart contracts variables from template:
 ::
 
-   cd ../gardener-smart-contracts
-   npm install
+  cd gardener-smart-contracts
+  make copy-env
 
-6. Migrate contracts to test blockchain
+Same attempt for gardener server:
+::
+
+  cd ../gardener-server
+  make copy-env
+
+
+Install dependencies
+--------------------
+
+Now, we are going to install dependencies that Gardener smart contract relies on. You can do it by:
+::
+
+  cd ../gardener-smart-contract
+  npm install
+
+
+Migrate contracts
+-----------------
+
+We are now able to migrate our contracts to test blockchain network
 ::
 
    npx truffle migrate --network ganache --reset
 
-7. Run server
+Start gardener server
+---------------------
+
+We are just one step behind sending sample request to blockchain network.
 ::
 
    cd ../gardener-server
```

Comment:
As before - I'd rather avoid the `cd` commands. I feel that this instruction also requires to jump between the directories a lot but maybe that's because of how it's presented here in the PR diff. I'd try to group this by the component:

1. Smart contracts - install dependencies, make env, run ganache, migrate
2. Server - install dependencies, make env, run
3. Monitor - install dependencies, make env, run
4. Sample request